api/api/admin.py

```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
-
+from api.models.usermodels import User
 # Register your models here.
+admin.site.register(User)
```

Comment:
Entiendo que esto se agregara para probar el token, pero no deberíamos subir al repositorio código del admin site de Django, esto porque nosotros estamos usando REST, todos los desarrollos en BE se deberían probar llamando directamente a los endpoints expuestos (entiendo que en este caso no se pudo probar de esa manera porque no tenemos un endpoint aun).